diamm/helpers/object_pagination.py

```diff
@@ -1,9 +1,9 @@
-from collections import OrderedDict
 import re
-
+from collections import OrderedDict
```

Comment:
`collections` is part of the standard library. 

https://docs.python.org/3/library/collections.html